Command/ListenCommand.php

```diff
@@ -105,12 +105,12 @@ private function start(ConnectionInterface $connection)
             $connection->getStreamingClient()->start();
         }
         catch (MemoryLimitException $e) {
-            $this->logger->critical($e->getMessage());
+            $this->logger->critical('ListenCommand-001 - '.$e->getMessage());
             $connection->getStreamingClient()->stop();
             return false;
         }
         catch (\Exception $e) {
-            $this->logger->critical($e->getMessage());
+            $this->logger->critical('ListenCommand-002 - '.$e->getMessage());
```

Comment:
This too with a different message.  Maybe "Unhandeled excception occurred" or "Listen Command: Unhandled exception" or something along those lines.  Please keep the exception message for both catch blocks